### DISCUSSION TITLE
Serverpod project without PostgreSql

### DISCUSSION THREAD

**Discussion by ketulrastogi** [serverpod/serverpod#612] [Q&A] [answered]
Is there any way to create a serverpod project that does not include PostgreSQL by default?
I want to use MongoDB as a Database in my project instead of Postgresql.

Allow users to select a database (**Postgresql** for now or **no database**) when creating a project using **serverpod create** command.

Thanks
Ketul

**Comment by vlidholt** [comment, marked as answer]
Currently, only Postgres is supported. We hope to add built-in support for more databases in the future, but there is nothing stopping you from also using MongoDB or any other database as long as there is a Dart package for it.

**Comment by perumalrollfi** [comment]
Hi @vlidholt,

Is there any dart package you recommend to use Serverpod with SQL server?

Thanks

**Reply by vlidholt** [reply]
Sorry, I haven't looked into that. Maybe someone else can help?

**Comment by yahu1031** [comment]
Packages like mysql_client, mysql1 will work. I have tried once. But it was a little hectic for me cause I'm not pretty good with sql stuff. But postgres is mandatory to have logs and metrics.
